### ISSUE TITLE
Os2

### ISSUE THREAD

**Issue by TeLLie** [sqlitebrowser/sqlitebrowser#2608]
Hi,
I have compile sqlite bowser to os2 using QT5 and gcc 9.2.0
Now a os2 user have this issue, and i hope this could be solved.

use SQLite DB Browser and would like to benefit from the improvements
introduced with the new version of SQLite ported to OS/2
(ArcaOS/eComStation). The current version on OS/2 seem to be 3.28.0

I read that you had to struggle to make DB Browser 3.9.1 work with QT
4.7.3, but would it be possible create a version that load whatever
sqlite3.dll is available?

I'd also would like to ask for support for national characters within DB 
Browser.
It can display national characters read from the database, but I can't
enter them in a query in the query editor.
It just show empty or some square/square with a ? depending on the font
i try to set/use.

![SQLiteDBBrowser](https://user-images.githubusercontent.com/1149380/110013464-f1429f00-7d21-11eb-94d6-533a364f63f6.png)

Hope to get any help on this 👍

**Comment by JanErikLaerka** [comment]
I can see the same thing in other textfields as one try to filter out information, while the database do show records with the characters (merged image) for the same view. Even the file browser is affected, but behave well with other QT-based applications. Text fields behave well with other applications and in the QT Designer.

**Comment by JanErikLaerka** [comment]
![DBBrowser](https://user-images.githubusercontent.com/13735819/110208692-f79f5b00-7e88-11eb-8d0e-caa12e044b78.png)

**Comment by justinclift** [comment]
Interesting.  Maybe some of the display widget's we're using aren't able to show the characters for some reason?

**Comment by JanErikLaerka** [comment]
It even affect the QT file browser with regular text fields throughout the application.
Is there some global project configuration that define the UI input character set?
At compile time or dynamically? Had the same issue in QT 4.x btw

Change of Preferences... to German + restart didn't help though, ü and Ü unreadable as well.

**Comment by JanErikLaerka** [comment]
![DBBrowserFileBrowser](https://user-images.githubusercontent.com/13735819/110232078-e6ebf500-7f1b-11eb-8044-9241ac96829b.png)

**Comment by justinclift** [comment]
Um, that screenshot is a bit confusing.  What is it showing?

**Comment by JanErikLaerka** [comment]
The frontmost File Browser show Virtual Box (QT 4.7.3) while the one behind show the file browser for DB Browser (QT 5.1.13)

**Comment by JanErikLaerka** [comment]
And here it is in DB Browser with QT 4.7.3
![DBBrowserFileBrowserQT473](https://user-images.githubusercontent.com/13735819/110234507-2d951b80-7f2b-11eb-8a16-650db6264510.png)

**Comment by JanErikLaerka** [comment]
Meant QT 5.13.1 above

**Comment by TeLLie** [comment]
Hi,
Is there found a solution for this yet, as it still prevents on OS2
Any thoughts?

Kind regards, Tellie

**Comment by justinclift** [comment]
Yeah, unfortunately @FriedrichFroebel is correct.  It'd need a developer with interest and access to OS/2 to track it down then figure out the fix.

That's not me personally, and I kind of doubt the other developers currently would be interested either.
